integrations/Bugsnag/browser.js

```diff
@@ -0,0 +1,59 @@
+import logger from "../../utils/logUtil";
+import  ScriptLoader  from "../ScriptLoader";
+import {
+  MAX_WAIT_FOR_INTEGRATION_LOAD,
+  INTEGRATION_LOAD_CHECK_INTERVAL
+} from "../../utils/constants";
+
+ class Bugsnag {
+  constructor(config) {
+    this.releaseStage = config.releaseStage;
+    this.apiKey = config.apiKey;
+    this.name = "BUGSNAG";
+    this.setIntervalHandler = undefined;
+  }
+
+   init() {
+    logger.debug("===in init Bugsnag===");
+    ScriptLoader(
+      "bugsnag-id",
+      "https://d2wy8f7a9ursnm.cloudfront.net/v6/bugsnag.min.js"
+    );
+
+     this.setIntervalHandler = setInterval(
+      this.initBugsnagClient.bind(this),
+      1000
+    );
+  }
+
+   initBugsnagClient() {
+    if (window.bugsnag !== undefined && window.bugsnag !== void 0) {
```

Comment:
what does void 0 does?
can we use a corresponding easy readable syntax

Reply:
Similarly it was done when initialising other destinations. To check if undefined or not.. Should I remove it ?